modules/importer.js

```diff
@@ -0,0 +1,31 @@
+import fs from "fs";
+
+export default class Importer {
+	import(filePath) {
+		return new Promise((resolve, reject) => {
+			fs.readFile(filePath, (err, buffer) => {
+				if (err) return reject(err);
+
+				return resolve(this.parseCsv(buffer));
+			});
+		});
+	}
+
+	importSync(filePath) {
+		try {
+			const buffer = fs.readFileSync(filePath);
+
+			return this.parseCsv(buffer);
+		} catch (err) {
+			console.log(err);
+		}
+	}
+
+	parseCsv(buffer) {
+		const data = buffer.toString().split(/\r\n|\r|\n/);
+
+		while (!data[data.length - 1]) data.pop();
+
+		return data.map(string => string.split(","));
```

Comment:
CSV parser should work like mentioned in a hometask ` (converting the data to JavaScript objects)`. So you should have then an array of objects with keys - first line elements in CSV file, and values - other lines accordingly to keys
E.g. assume you have CSV like

**NAME, SURNAME**
**Alex, Bruzgin**
**Mikhail, Yamayeu**

Then you should have valid JSON like `[{"NAME": "Alex", "SURNAME": "Bruzgin"}, {"NAME": "Mikhail", "SURNAME": "Yamayeu"}]`